recipe/meta.yaml

```
@@ -1,45 +1,54 @@
 {% set name = "streamlit-aggrid" %}
-{% set version = "0.3.3" %}
+{% set version = "0.3.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d96edb102df98450f3b6bcf5a52b64e6ceef9678e82e2dba9cfef074e7f134ab
+  sha256: d4b776e4e7c2a91808744873843c8236065e263c22cc39e7267383ccc81486a3
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
   number: 0
+  # poetry not available for py312
+  # streamlit not available for s390x
+  skip: true  # [py<38 or py>311 or s390x]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
     - poetry
-    - python >=3.7,<4.0
+    - python
   run:
+    - python
```

according to poetry dependencies `!=3.9.7`
https://github.com/PablocFonseca/streamlit-aggrid/blob/e673472b9298cc62f212466404d9256a6a020cb5/pyproject.toml#L14

---

```suggestion
    - python !=3.9.7
```

good catch thank you!

---

I'm not sure if this is a legit way of skipping this particular version of python or I shall use a selector on `skip: true` but which selector? I've never seen something like `skip: true  # [py==397]`

---

I would leave it like you did. The `skip` statement should not be using because you're not avoiding the build for that python version, you are simply restricting the python environment when it's solved upon installation.